.drone.yml

```diff
@@ -37,7 +37,7 @@ steps:
     pull: always
     image: golang:1.15
     commands:
-      - make golangci-lint vet
+      - make golangci-lint
```

Comment:
```suggestion
      - make golangci-lint
      - go vet ./....
```

Or split make target "vet" ...

Reply:
It's unnecessary, we have did that on before step.

Reply:
but this should -go vet- the windows code too ...

Reply:
It's unnecessary. We have done that on non windows lint.